src/utils/semantics.utils.ts

```diff
@@ -0,0 +1,31 @@
+import { blanksDefaultTranslations, defaultTranslations, dragTextDefaultTranslations } from '../constants/defaultTranslations';
+import { AnswerModeType, type Params } from '../types/types';
+
+export const getDefaultParams = (): Required<Params> => {
+  return {
+    description: undefined,
+    sourceLanguage: 'en',
+    targetLanguage: 'nb',
+    // @ts-expect-error - overallFeedback type is incorrect
```

Comment:
h5p-types is now updated to 5.2.0, and optional fields can now be undefined. But if you want these fields to be completely optional, further changes must be made in h5p-types. I have notified Sindre about this.

Reply:
Ideally `h5p-types` should reflect the true type requirements. Optional fields in semantics should not be required to be set at all. But since that's a shortcoming upstream that you needed to work around, please add a respective comment and we're good to go.